docker/Makefile.core.jenkins

```diff
@@ -0,0 +1,22 @@
+.PHONY: clean coldstart mrproper up
+all: up
+
+compose := docker-compose \
+	-f docker-compose.lisk.core.yml \
+	-f docker-compose.yml \
```

Comment:
I would be in favour of moving the file to the `lisk_core` directory and renaming the file to something like `docker-compose.core.yml` to maintain a clear separation.

```suggestion
	-f docker-compose.yml \
	-f lisk_core/docker-compose.core.yml \
```